packages/web-components/src/accordion/fixtures/base.html

```diff
@@ -1,7 +1,7 @@
 <fluent-design-system-provider use-defaults>
   <style>
-    .icon {
-      stroke: var(--accent-fill-rest);
+    fluent-accordion-item::part(icon) {
+      fill: var(--accent-fill-rest);
```

Comment:
@khamudom If this style is being added to accommodate the design, I think this should be in the `style` file - this is just for the examples.

Reply:
good point. This can be moved to `style`.

Reply:
updated